.github/workflows/release-master.yaml

```diff
@@ -1,27 +1,104 @@
-name: Latest Release
+name: Build and Push Image
 
 on:
   push:
     branches:
       - "main"
-    paths-ignore:
-      - "**/*.png"
-      - "**/*.svg"
-      - "**/*.yaml"
-      - "**/*.md"
-
+    tags:
+      - "v*.*.*"
```

Comment:
Can't kmesh's TAG cancel the 'v'?

Reply:
maybe a separate issue, why strip `v` prefix

Reply:
Because there is a PR in the Kurator community for removing the prefix `v`
https://github.com/kurator-dev/kurator/pull/433

Reply:
Please take alook https://github.com/orgs/kmesh-net/packages/container/package/kmesh